ante/ante.go

```diff
@@ -47,20 +48,19 @@ func NewAnteHandler(opts HandlerOptions) (sdk.AnteHandler, error) {
 
 	anteDecorators := []sdk.AnteDecorator{
 		ante.NewSetUpContextDecorator(), // outermost AnteDecorator. SetUpContext must be called first
-		// ante.NewExtensionOptionsDecorator(opts.ExtensionOptionChecker),
+		ante.NewRejectExtensionOptionsDecorator(),
 		ante.NewValidateBasicDecorator(),
 		ante.NewTxTimeoutHeightDecorator(),
 		ante.NewValidateMemoDecorator(opts.AccountKeeper),
 		ante.NewConsumeGasForTxSizeDecorator(opts.AccountKeeper),
 		gaiafeeante.NewFeeDecorator(opts.BypassMinFeeMsgTypes, opts.GlobalFeeSubspace, opts.StakingSubspace),
-		// if opts.TxFeeCheck is nil,  it is the default fee check
 		ante.NewDeductFeeDecorator(opts.AccountKeeper, opts.BankKeeper, opts.FeegrantKeeper),
-		// ante.NewDeductFeeDecorator(opts.AccountKeeper, opts.BankKeeper, opts.FeegrantKeeper, opts.TxFeeChecker),
 		ante.NewSetPubKeyDecorator(opts.AccountKeeper), // SetPubKeyDecorator must be called before all signature verification decorators
 		ante.NewValidateSigCountDecorator(opts.AccountKeeper),
 		ante.NewSigGasConsumeDecorator(opts.AccountKeeper, sigGasConsumer),
 		ante.NewSigVerificationDecorator(opts.AccountKeeper, opts.SignModeHandler),
 		ante.NewIncrementSequenceDecorator(opts.AccountKeeper),
+		ibcante.NewAnteDecorator(opts.IBCkeeper),
 		// TODO: check what this is
 		// ibcante.NewRedundantRelayDecorator(opts.IBCkeeper),
```

Comment:
I think `NewRedundantRelayDecorator` is the API for ibc v5...probably need to revert to old API https://github.com/cosmos/ibc-go/blob/ed71af861e6dfe5fa5d942fbdc7eaa5e88767727/docs/migrations/v4-to-v5.md#ante-decorator

Reply:
`ibcante.NewAnteDecorator(opts.IBCkeeper)` is for sdk0.45 and ibc v4, right ?

Reply:
Yeah so we can just remove the commented out line and we're good to go.